tree/30_generic_methods/_service_check_running_smf.cf

```diff
@@ -36,14 +36,19 @@ bundle agent _service_check_running_smf(service_name)
       "class_prefix"        string => canonify(join("_", "promisers"));
       "args"                 slist => { "${service_name}" };
 
-  methods:
 
+  classes:
+      "report_mode"    expression => "${report_data.should_report}";
```

Comment:
Could the name of the class be boolean like `should_report` or `reports_enabled`? report_mode means nothing.

Reply:
Caution, we have "full_compliance", "changes_only" and "reports_disabled" asglobal classes for reporting - we should not use a name too close to these naming, to avoid us major headache.

Reply:
should_report sounds correct - it matches the name of the variable, so it's easily grepable